lib/datastore_v1.proto

```diff
@@ -180,6 +180,14 @@ message Entity {
   // The entity's properties.
   // Each property's name must be unique for its entity.
   repeated Property property = 2;
+
+  // The entity's properties.
+  // The map's keys are property names.
+  // A property name matching regex `__.*__` is reserved.
+  // A reserved property name is forbidden in certain documented contexts.
+  // The name must not contain more than 500 characters.
+  // The name cannot be `""`.
+  map<string, Value> properties = 3;
```

Comment:
I don't think we want to change the protocol buffer file for the v1beta2 implementation; I pulled ours straight from [the official one](https://github.com/GoogleCloudPlatform/google-cloud-datastore/blob/v1beta2-rev1-3.0.2/proto/datastore_v1.proto), and I think we should stick with it.

When we're ready to upgrade to v1beta3, we have the protobuf file for that in a separate branch, here: https://github.com/peburrows/diplomat/blob/v1beta3/lib/datastore_v1beta3.proto